proposals/first-class-span-types.md

```diff
@@ -472,6 +470,33 @@ list.RemoveAll(toRemove.Contains); // error CS1113: Extension method 'MemoryExte
 
 The break will be mitigated by not considering span conversions for extension receiver in method group conversions.
 
+### Ignoring more user-defined conversions
+
+We defined a set of type pairs for which there are language-defined implicit and explicit span conversions.
+Whenever a span conversion exists from `T1` to `T2`, any user-defined conversion from `T1` to `T2` is [ignored][udc]
```

Comment:
> Whenever a span conversion exists ...

I assume this means span conversions in the language correct? If so should we specify that here?